packages/components/src/tabs/README.md

```diff
@@ -1,254 +1,216 @@
 # Tabs
```

Comment:
Repeated from https://github.com/WordPress/gutenberg/pull/68249/files#r1895926109 — `Tabs` is currently a private API, but there's no mention of this in the README. Should we add a way to flag that in the generated README?

Reply:
Proposed in #68317